src/main/pg/gyrodev.h

```diff
@@ -34,7 +35,8 @@ typedef struct gyroDeviceConfig_s {
     uint8_t i2cBus;
     uint8_t i2cAddress;
     ioTag_t extiTag;
-    uint8_t align;        // sensor_align_e
+    uint8_t alignment;        // sensor_align_e
```

Comment:
It would be great to remove legacy alignment (eventually) ...

Reply:
yes, initially I did in the other PR, and then had to add it back for MSP compatibility :(  This is something that we need a plan for if we're going to do it without causing MSP client authors headaches.